### PR TITLE
Adds tag to APIs

### PR DESCRIPTION
**Pull request by gildub**
All APIs updated to following:
*"Alarming API Legacy"
*"Block Storage API Legacy"
*"Metering API Legacy"

*"Application Catalog API Reference 5.0.0"
*"Backup and Disaster Recovery Service API Reference 5.0.1"
*"Block Storage API Reference 12.0.0"
*"Clustering API Reference 4.0.1"
*"Compute API Reference 17.0.0"
*"Container Infrastructure Management API Reference 5.0.1"
*"Data Processing API Reference 8.0.0"
*"Data Protection API Reference 0.5.1"
*"Database API Reference 9.0.0"
*"Designate API Reference 6.0.0"
*"Identity API Reference v3.8"
*"Identity API Reference v3.8"
*"Image Service API Reference 16.0.0"
*"Image Service API Reference 16.0.0"
*"Ironic API Reference 9.2.1"
*"Messaging Service API Reference 6.0.0"
*"Networking API Reference 1.10.1"
*"NFV Orchestration API Reference 0.8.1"
*"Object Storage API Reference 2.16.1"
*"Octavia API Reference 2.0.0"
*"Orchestration API Reference 10.0.0"
*"Search Service API Reference 3.0.1"
*"Shared File Systems API Reference 6.0.0"

### CODE REVIEW

File: lib/misty/openstack/ironic/ironic_v1.rb [hunk 1]
```diff
@@ -26,14 +30,14 @@ def api
   {:GET=>[:get_console], :PUT=>[:start_stop_console]},
  "/v1/nodes/{node_ident}/vendor_passthru/methods"=>{:GET=>[:list_methods]},
  "/v1/nodes/{node_ident}/vendor_passthru?method={method_name}"=>
-  {:GET=>[:call_a_method]},
+  {:METHOD=>[:call_a_method]},
```
Comment (andypf) [inline lib/misty/openstack/ironic/ironic_v1.rb:33]: Is that correct?

File: lib/misty/openstack/ironic/ironic_v1.rb [hunk 2]
```diff
@@ -51,14 +55,33 @@ def api
...
  "/v1/drivers"=>{:GET=>[:list_drivers]},
  "/v1/drivers/{driver_name}"=>{:GET=>[:show_driver_details]},
  "/v1/drivers/{driver_name}/properties"=>{:GET=>[:show_driver_properties]},
  "/v1/drivers/{driver_name}/raid/logical_disk_properties"=>
   {:GET=>[:show_driver_logical_disk_properties]},
- "/v1/drivers/{driver_name}/vendor_passthru/methods"=>{:GET=>[:list_driver_method]},
+ "/v1/drivers/{driver_name}/vendor_passthru/methods"=>{:GET=>[:list_driver_methods]},
  "/v1/drivers/{driver_name}/vendor_passthru?method={method_name}"=>
-  {:GET=>[:call_a_driver_method]},
+  {:METHOD=>[:call_driver_method]},
```
Comment (andypf) [inline lib/misty/openstack/ironic/ironic_v1.rb:84]: Same here.

Comment (gildub) [reply lib/misty/openstack/ironic/ironic_v1.rb:84]: same

File: lib/misty/openstack/ironic/ironic_v1.rb (hunk 1 above)
Comment (gildub) [reply lib/misty/openstack/ironic/ironic_v1.rb:33]: To be replaced by GET

**Comment by gildub** [comment]
That should be GET. Thanks fixed with https://github.com/flystack/misty/compare/master...gildub:no-generic-method

**Pull request merged**